src/map/clif.c

```diff
@@ -16163,14 +16273,32 @@ void clif_parse_cz_config(int fd, struct map_session_data *sd)
 	int type = RFIFOL(fd, 2);
 	int flag = RFIFOL(fd, 6);
 
-	if (type == CZ_CONFIG_OPEN_EQUIPMENT_WINDOW) {
+	switch(type) {
+	case CZ_CONFIG_OPEN_EQUIPMENT_WINDOW:
 		sd->status.show_equip = flag;
-	} else if (type == CZ_CONFIG_HOMUNCULUS_AUTOFEEDING) {
+		break;
+
+	case CZ_CONFIG_PET_AUTOFEEDING: {
+		struct pet_data *pd;
```

Comment:
because between this two lines no code, it can be joined into one ``struct pet_data *pd = sd->pd``